payments-core/res/values/totranslate.xml

```diff
@@ -4,5 +4,16 @@
     Items here need to be entered in our translation tool
     -->
     <!-- Affirm buy now, pay later copy -->
-    <string name="affirm_buy_now_pay_later">Buy now, pay later with Affirm.</string>
+    <string name="affirm_buy_now_pay_later">Buy now, pay later with</string>
+    <!-- An error message displayed when the customer's BSB number is incomplete. -->
+    <string name="bsb_incomplete">The BSB number you entered is incomplete.</string>
+    <!-- An error message displayed when the customer's BSB number is invalid. -->
+    <string name="bsb_invalid">The BSB number you entered is invalid.</string>
+    <!-- AU BECS mandate text -->
+    <string name="au_becs_mandate">By clicking Pay, you agree to this Direct Debit Request and the <a href="https://stripe.com/au-becs-dd-service-agreement/legal">Direct Debit Request service agreement</a>, and authorise Stripe Payments Australia Pty Ltd ACN 160 180 343 Direct Debit User ID number 507156 (“Stripe”) to debit your account through the Bulk Electronic Clearing System (BECS) on behalf of %s (the “Merchant”) for any amounts separately communicated to you by the Merchant. You certify that you are either an account holder or an authorised signatory on the account listed above.</string>
```

Comment:
Note to self: this string is translated in stripe-js code base.

Reply:
I ended up splitting the mandate string into three strings (pre-link, link, and post-link) because I couldn't figure out a good way to simply parse the string and insert the link where it belonged. But this solution also doesn't feel great, especially when the strings have to be translated at some point

Reply:
sounds like maybe we can file this and bruno will address it in a future PR?